docs/API.md

```diff
@@ -1006,36 +1007,36 @@ bucketNotification := BucketNotification{}
 bucketNotification.AddTopic(topicConfig)
 err := c.SetBucketNotification(bucketName, bucketNotification)
 if err != nil {
-	fmt.Println("Cannot set the bucket notification: " + err)
+    fmt.Println("Cannot set the bucket notification: " + err)
 }
 ```
 
 <a name="RemoveAllBucketNotification"></a>
 ### RemoveAllBucketNotification(bucketName string) error
 
-Remove all configured bucket notifications on a bucket.
+Remove all configured notifications on a bucket.
```

Comment:
"Remove all configured bucket notifications" is correct..  "notifications" what is sent by the server.